test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerProcessTests.cs

```diff
@@ -75,7 +93,7 @@ public void Constructor_RegistersHostStartedEvent()
                 return eventType == typeof(HostStartEvent);
             };
 
-            _eventManager.Verify(_ => _.Subscribe(It.Is<IObserver<ScriptEvent>>(p => validate(p))), Times.Once());
+            _eventManager.Verify(_ => _.Subscribe(It.Is<IObserver<ScriptEvent>>(p => validate(p))), Times.AtLeastOnce());
```

Comment:
Why this change?

Reply:
Subscribe is called twice due to addition of `_rpcWorkerProcessWithExecutablePath ` in ctor

Reply:
This invalidates the test as it wouldn't catch a regression that ends up resulting multiple events. Can you update this to use `Exactly` instead?

Refactoring this test to avoid the code duplication with the new instance of the `RpcWorkerProcess` would be preferred over this, though.